src/main/java/it/gov/pagopa/reportingorgsenrollment/config/ResponseValidator.java

```diff
@@ -0,0 +1,54 @@
+package it.gov.pagopa.reportingorgsenrollment.config;
+
+import java.util.Set;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+
+import org.apache.commons.lang3.StringUtils;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.AfterReturning;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import it.gov.pagopa.reportingorgsenrollment.exception.AppException;
+
+@Aspect
+@Component
+public class ResponseValidator {
```

Comment:
This class is useful for validating the data in the database (If the constraints in the DB is not good). I think we can remove this class in this repo